src/components/ReportActionItem/ReportActionItemImage.tsx

```diff
@@ -116,33 +128,42 @@ function ReportActionItemImage({
         );
     }
 
-    const attachmentModalSource = tryResolveUrlFromApiRoot(image ?? '');
+    const originalImageSource = tryResolveUrlFromApiRoot(image ?? '');
     const thumbnailSource = tryResolveUrlFromApiRoot(thumbnail ?? '');
     const isEReceipt = transaction && !hasReceiptSource(transaction) && hasEReceipt(transaction);
+    const isPDF = filename && Str.isPDF(filename);
```

Comment:
Is there no helper method that could be reused for this? Feels like we should extract